tier2/{{cookiecutter.project_slug}}/.github/workflows/contributors.yml

```diff
@@ -0,0 +1,63 @@
+on:
+    push:
+        branches:
+            - main
+    workflow_dispatch:
+
+jobs:
+    update-contributors:
+        runs-on: ubuntu-latest
+        name: Update contributors info in MAINTAINERS.md
+        permissions:
+            contents: write
+            pull-requests: write
+        steps:
+            # Update contributor list
+            - name: Contribute List
+              uses: akhilmhdh/contributors-readme-action@v2.3.10
+              env:
+                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+              with:
+                  # https://github.com/marketplace/actions/contribute-list#optional-parameters
+                  readme_path: MAINTAINERS.md
+                  use_username: false
+                  commit_message: "BOT: Update contributors info in MAINTAINERS.md"
+
+            # Update contributor count
+            - name: Checkout repository
+              uses: actions/checkout@v4
+
+            - name: Pull changes from contributors-readme-action
+              run: |
+                  git pull
+
+            - name: Get repository contributors count
+              id: get_contributors
+              # https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#list-repository-contributors
+              # https://docs.github.com/en/graphql/reference/objects#repositorycollaboratorconnection
+              # https://docs.github.com/en/graphql/guides/forming-calls-with-graphql#communicating-with-graphql
+              # CANNOT have newlines!
+              run: |
+                  TEMPLATE_QUERY='query { repository(owner: \"OWNER\", name: \"REPO\") { collaborators { totalCount } } }'
+                  OWNER=$(echo $GITHUB_REPOSITORY | cut -d'/' -f1)
+                  REPO=$(echo $GITHUB_REPOSITORY | cut -d'/' -f2)
+                  QUERY=$(echo $TEMPLATE_QUERY | sed "s/OWNER/$OWNER/" | sed "s/REPO/$REPO/")
```

Comment:
Looks good although something like 
```bash
QUERY='query { repository(owner\"$OWNER\", name: \"$REPO\") { collaborators { totalCount } } } '
```
after the REPO and OWNER env variables are declared might be smoother. This isn't a blocking concern though.

Reply:
Fixed!